.github/workflows/makefile-test.yaml

```diff
@@ -21,8 +21,25 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          brew install scdoc mandoc coreutils ninja cbmc aws/tap/cbmc-viewer litani
-          python3 -m pip install pyyaml jinja2
+          get_latest_release() {
+            curl --silent "https://api.github.com/repos/$1/releases/latest" |
+            jq '.tag_name' | tr -d '"'
```

Comment:
Consider `jq -r '.tag_name'` instead of piping into tr.  This is short for `--raw-output` printing strings without json quotation.